shared/domain/src/main/java/com/ivy/domain/features/IvyFeatures.kt

```diff
@@ -67,6 +67,13 @@ class IvyFeatures @Inject constructor() : Features {
         defaultValue = false
     )
 
+    override val showAccountColorsInTransactions = BoolFeature(
+        key = "show_account_color",
+        group = FeatureGroup.Other,
+        name = "Show Account Color",
+        description = "Show account specific colors for accounts in transactions "
```

Comment:
Let's be explicit and add `defaultValue = false`

Reply:
Done